torch/_dynamo/variables/torch.py

```diff
@@ -1325,15 +1325,35 @@ def patched_fn(*args, **kwargs):
         # variant torch ops, the original function could come from a user
         # defined `@allow_in_graph` function as well, which doesn't have the
         # same semantics as the torch ops.
-        fake_out_shape = None
-        if "out" in kwargs and isinstance(kwargs["out"], variables.TensorVariable):
-            # Calling fake tensor propagation can mutate the out= tensor in
-            # tx.output.tracked_fakes. tracked_fakes are used to apply
-            # symbolic_shape guards. Mutating them destroys the information
-            # prior to tracing, which is essential for creating right
-            # guards. So save the shape now, and check later if it has
-            # changed. If it has, graph break.
-            fake_out_shape = kwargs["out"].proxy.node.meta["example_value"].shape
+
```

Comment:
Can you explain a bit about why this changed?

Reply:
Oops forgot.

Basically all the code here handles `torch.some_op(..., out=...)`, and in principle we only have 2 cases -- `out` is a tensor or a tuple/list of tensors, and we just need to check and graph break on contiguity/resize.

But the old way was a bit convoluted for historical reasons, and the special branch to handle `torch.some_op(..., out=...)` _returning_ `None` forgot to check for resize, and my change exposed this bug: https://github.com/pytorch/pytorch/blob/4515aaa7cc6fed3c9908586dab105b96c114d185/torch/_dynamo/variables/torch.py#L1389-L1408

So I just went and refactored the code dup and fixed the bug here.